src/ipc/ipc-common.c

```diff
@@ -317,6 +336,11 @@ static enum task_state ipc_do_cmd(void *data)
 {
 	struct ipc *ipc = data;
 
+#ifdef CONFIG_SOF_TELEMETRY_IO_PERFORMANCE_MEASUREMENTS
+	/* Increment performance counters */
+	io_perf_monitor_update_data(ipc->io_perf_in_msg_count, 1);
```

Comment:
why IPC commands are counted for performance? In some use-cases there are many IPCs, in others none, like plain playback or capture?

Reply:
In ref. FW this feature did also count some events like IPC IDC and some interrupts.